### PR TITLE
ov traveltime map

### PR DESCRIPTION
**Pull request by CBAmsterdam**

### CODE REVIEW

File: ovtraveltime.map [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    VALIDATION
+       # %scraped_at% substitutions can only have letters, digits and underscore
+       'default_scraped_at' '2019-04-03' 
+       'scraped_at'     '^[0-9]{4}-[0-9]{2}-[0-9]{2}$'
+    END  
+
+    DATA            "geo_location FROM (select * from public.sum_ovkv6_speed where vehicle_date='%scraped_at%') as subquery USING srid=4326 USING UNIQUE id"
+    TYPE            LINE
+    MINSCALEDENOM   100
+    MAXSCALEDENOM   400001
```
Comment (RRMoelker) [inline ovtraveltime.map]: 400001 intentional? Or should it be 40000

File: ovtraveltime.map [hunk 2]
```diff
@@ -0,0 +1,70 @@
+MAP
+  NAME "TRAVELTIME"
+  INCLUDE "header.inc"
+
+  WEB
+    METADATA
+      "ows_title"    "traveltime"
+      "ows_abstract" "Traveltime OV visualization"
+      "wms_extent"          "100000 450000 150000 500000"
```
Comment (RRMoelker) [inline ovtraveltime.map]: indent

File: ovspeed_classes.map [hunk 3]
```diff
@@ -0,0 +1,95 @@
+CLASS
```
Comment (aditudorache) [inline ovspeed_classes.map]: don't use map extension for includes, use .inc extension 
don't place the inc files in the root of the project, create an includes folder and place the *.inc files in a folder that matches the map name.
please correct this for speed_classes.map as wel

File: includes/speed_classes.inc [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  EXPRESSION (([velocity] >= 160) AND ("[road_type]" = "H"))
+  STYLE
+    WIDTH 10
+    OUTLINECOLOR "#FF00FF"
+    LINECAP BUTT
+    LINEJOIN MITER
+  OPACITY 50   
+  END
+END
+#=============================================================================
```
Comment (RRMoelker) [inline includes/speed_classes.inc]: Merge conflict left overs?

File: ovtraveltime.map [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+    METADATA
+      "ows_title"           "latest_traveltime"
+      "ows_group_title"     "latest_traveltime"
+      "ows_abstract"        "latest traveltime per ov regio Amsterdam"
+      "gml_featureid"       "ID"
+      "gml_include_items"   "all"
+      "gml_types"           "auto"
+    END
+    INCLUDE         "includes/ovspeed_classes.inc"
```
Comment (RRMoelker) [inline ovtraveltime.map]: This `includes` folder seems like an anti-pattern. But out of scope for now

Comment (yelsherbini) [reply ovtraveltime.map]: Are you referring to the name `includes` or the folder itself?

**Pull request merged**